shared_model/validators/block_variant_validator.hpp

```diff
@@ -20,7 +20,8 @@ namespace shared_model {
         : public shared_model::validation::AbstractValidator<
               shared_model::interface::BlockVariant> {
      public:
-      Answer validate(const shared_model::interface::BlockVariant &m) override {
+      Answer validate(
```

Comment:
Do we still need block variant validator?

Reply:
It will be removed in next PR as soon as EmptyBlock will be removed completely